web/templates/layout/app.html.eex

```diff
@@ -62,8 +62,16 @@
         <main id="body">
           <section class="h-100 pa3 pa5-ns bt b--black-10 bg-white">
             <section class="mw7 center">
-              <p class="" role="alert"><%= get_flash(@conn, :info) %></p>
-              <p class="ma3 ma4-m ma4-l" role="alert"><%= get_flash(@conn, :error) %></p>
+              <%
+                info = get_flash(@conn, :info)
+                error = get_flash(@conn, :error)
+              %>
+              <%= if info do %>
+                <div class="b--hl-yellow br3 hl-bg-success center tc pv2" role="alert"><%= get_flash(@conn, :info) %></div>
+              <% end %>
+              <%= if error do %>
```

Comment:
my preference would be
```if get_flash(@conn, :info) do```
Unless we're going to be using info more than once, but more of a stylistic thing tbh